### DISCUSSION TITLE
Point2Grid with Docker

### DISCUSSION THREAD

**Discussion by spence98** [dtcenter/METplus#1234] [File I/O] [answered]
Hello, this is Michelle again. I'm opening a new ticket for a different (but new) error/issue.

I am trying to process LSRs using MET with a Docker. I am able to run the ascii2nc command and successfully produce a .nc file that does have data within it. However, when I try to run point2grid, via the command example in the MET userguide (see below), it is not working. From my understanding, I shouldn't need to pass any configuration file in when just using the point2grid command, so I honestly don't know what is occurring or how to get past the issue. 

![point_to_grid_error](https://user-images.githubusercontent.com/90273095/139778939-2dd49787-edc6-4391-a44e-9f0d47bea264.png)
.

**Comment by jprestop** [comment, marked as answer]
Hi Michelle.  Could you please send me your data see [How to Send Us Data](https://github.com/dtcenter/METplus/discussions/954) so that I can take a look at try to reproduce the problem?  Thanks!  (Side note:  We have received the messages in the other ticket and will follow up shortly.)

Julie

**Reply by jprestop** [reply]
Hi Michelle (@spence98).  I just wanted to follow up and see if you still needed assistance with this issue?

**Reply by spence98** [reply]
Hello Julie! I am sorry for my delayed response. I tried to send the data according to the directions, but I wasn't able to get it to work, so I'm attaching the .nc file as a zipped file here. 

I will say, I did have success with processing the LSRs over the weekend using the python wrapper (I literally did nothing different, but all the sudden it worked). So, in terms of this specific question, I am no longer in need of assistance, however I wanted to provide the data to you in case you wanted to try to recreate the error or look into things for future development!

[7mar08_stormreps.nc.zip](https://github.com/dtcenter/METplus/files/7509129/7mar08_stormreps.nc.zip)

**Reply by jprestop** [reply]
Thank you for providing this sample data.  I was able to reproduce the problem that you experienced.  The issue lies with the way the field was being specified.  It needs to have a "name" and "level" setting in there, although I see our [Point2Grid documentation](https://met.readthedocs.io/en/develop/Users_Guide/reformat_point.html?highlight=point2grid#required-arguments-for-point2grid) does not clearly state that so we'll work on updating that with this [Issue](https://github.com/dtcenter/MET/issues/1962).

```
point2grid 7mar08_stormreps.nc G211 7mar08_point2grid.nc -field 'name="Fscale"; level="*,*";'
```

We've found that it's difficult to track new posts to previously "Answered" discussions. So at this point, I'm going to "Lock" this Discussion to prevent future posts. That's how we encourage users to open new discussions for new questions. Feel free to do so if more questions arise.
